lib/LANraragi/Plugin/Metadata/Eze.pm

```diff
@@ -115,6 +128,30 @@ sub tags_from_eze_json {
     my $site   = $hash->{"gallery_info"}->{"source"}->{"site"};
     my $gid    = $hash->{"gallery_info"}->{"source"}->{"gid"};
     my $gtoken = $hash->{"gallery_info"}->{"source"}->{"token"};
+    my $category = $hash->{"gallery_info"}->{"category"};
+    my $uploader = $hash->{"gallery_info_full"}->{"uploader"};
+    my $timestamp = $hash->{"gallery_info_full"}->{"date_uploaded"};
+
+    if ( $timestamp ) {
+        # convert microsecond to second
+        $timestamp = $timestamp / 1000;
+    } else {
+        my $upload_date = $hash->{"gallery_info"}->{"upload_date"};
+        my $time = timegm_posix($$upload_date[5],$$upload_date[4],$$upload_date[3],$$upload_date[2],$$upload_date[1]-1,$$upload_date[0]-1900);
+        $timestamp = $time;
+    }
+
+    if ( $category ) {
```

Comment:
My idea is that user can use filters to block them if they like. Also, the E-Hentai plugin has this  tag

https://github.com/Difegue/LANraragi/blob/dev/lib/LANraragi/Plugin/Metadata/EHentai.pm#L273

I think add it can make these two plugins more consistent.

Reply:
Oh right, didn't see `category` was also outside of the additional tags in the EH plugin -- Combined with the fact that you can indeed use dynamic categories to map back to EH category tags, I guess this makes for a good argument. 😅